specification/hugr.md

```diff
@@ -1239,8 +1236,13 @@ The `Replace` method takes as input:
   - a list $\mu\_\textrm{inp}$ of `NewEdgeSpec` which all have their `TgtNode`in
     $G$ and `SrcNode` in $\Gamma \setminus R$;
   - a list $\mu\_\textrm{out}$ of `NewEdgeSpec` which all have their `SrcNode`in
-    $G$ and `TgtNode` in $\Gamma \setminus R$ (where `TgtNode` and `TgtPos` describe
-    an existing incoming edge of that kind from a node in $R$).
+    $G$ and `TgtNode` in $\Gamma \setminus R$, where `TgtNode` and `TgtPos` describe
+    an existing incoming edge of that kind from a node in $R$.
+  - a list $\mu\_\textrm{new}$ of `NewEdgeSpec` which all have both `SrcNode` and `TgtNode`
+    in $\Gamma \setminus R$, where `TgtNode` and `TgtPos` describe an existing incoming
+    edge of that kind from a node in $R$.
+
+Note that for `NewEdgeSpec`s with `EdgeKind` == `Value`, the `TgtNode` + `TgtPos`s will all be unique when considering all three lists together, and similarly for `EdgeKind` == `Static`.
```

Comment:
Of course, "when considering all three lists together" implies that there *is* some idea of considering the targets of `mu_inp` (in G) as distinct from the targets of `mu_out` and `mu_new`, which I claimed was not a good idea in my description....so I could say "both in mu_inp, and (mu_out + mu_new together)"....